scripts/find_spammy_content.py

```diff
@@ -70,7 +70,7 @@ def find_spammy_content(regex, days, model, ban):
             item_data['fullname'] = item.creator.fullname
             data.append(item_data)
             if ban:
-                item.suspend_spam_user(item.creator, train_akismet=False)
+                item.suspend_spam_user(item.creator)
```

Comment:
I think that not training akismet was intentional in these cases, since we wanted to avoid training on broad false positives. Just because a user is Spammy doesn't mean that everything they submitted should be considered spammy.

Reply:
Yeah the reason I'm not passing the kwarg here is because suspending the user never trains the spam filter anymore.